server/aws/lambda/aggregate_values.js

```diff
@@ -137,15 +236,27 @@ function sendToDeadLetterQueue(payload, err) {
 }
 
 exports.handler =  (event, context, callback) => {
-    const aggregates = aggregateEvents(event);
-    for (const aggregate in aggregates) {
-        const payload = generatePayload(aggregates[aggregate]);
-        documentClient.update(payload, (err, data) => { 
-            if (err){ 
-                console.error(`Failed updating sending to Dead Letter Queue ${err}`);
-                sendToDeadLetterQueue(payload, err);
-            }
-        });
+
+    try {
+
+        const aggregates = aggregateEvents(event);
+        for (const aggregate in aggregates) {
+            const payload = generatePayload(aggregates[aggregate]);
+            documentClient.update(payload, (err, data) => {
+                if (err){
+                    console.error(`Failed updating sending to Dead Letter Queue ${err}`);
+                    sendToDeadLetterQueue(payload, err);
+                }
+            });
+        }
+
+    } catch(err){
+
+        //TODO: send to s3 bucket
+        console.error(`failed event: ${JSON.stringify(err)}`);
```

Comment:
Not knowing what you are getting back here, but noting inconsistency with usage of err in the generatePayload exception handler.

Reply:
Also removed so we aren't logging out things we shouldn't be.